pilot/pkg/features/pilot.go

```diff
@@ -422,6 +432,9 @@ var (
 		"If enabled, addition runtime asserts will be performed. "+
 			"These checks are both expensive and panic on failure. As a result, this should be used only for testing.",
 	).Get()
+
+	TLSInboundCipherSuites = env.RegisterStringVar("TLS_INBOUND_CIPHER_SUITES", DefaultInboundCiphers,
```

Comment:
Why is this command line and not in MeshConfig?

Reply:
This is an env var btw, but I agree mesh config (or proxy config?) is the right place. Probably meshConfig

Reply:
Thanks! I will move it to MeshConfig.

Reply:
@louiscryan https://github.com/istio/api/pull/1912 is the PR for making it an API in MeshConfig. Please review it.